Resources/Private/GraphQL/schema.root.graphql

```diff
@@ -48,54 +48,42 @@ type Query {
 
 type Mutation {
     updateAsset(
-        id: AssetId!,
-        assetSourceId: AssetSourceId!,
-        label: String,
-        caption: String,
+        id: AssetId!
+        assetSourceId: AssetSourceId!
+        label: String
+        caption: String
         copyrightNotice: String
     ): Asset!
 
-    deleteAsset(
-        id: AssetId!,
-        assetSourceId: AssetSourceId!
-    ): Boolean!
+    deleteAsset(id: AssetId!, assetSourceId: AssetSourceId!): Boolean!
 
-    tagAsset(
-        id: AssetId!,
-        assetSourceId: AssetSourceId!,
-        tag: TagLabel!
-    ): Asset!
+    tagAsset(id: AssetId!, assetSourceId: AssetSourceId!, tag: TagLabel!): Asset!
```

Comment:
Any reason why you changed all the formatting here?
The multilines were intentional.

Reply:
VSC/prettier or whatever else autoformats it.
Let's not do manual formatting, please, I hate overriding what VSC wants to do...

Reply:
oh interesting, it didn't do that for me and I didn't know it applies to this filetype.

Reply:
I have graphql plugin installed in VSC, maybe it does it?

Reply:
and it didn't change that for the `updateAsset` mutation?

Reply:
apparently it'd go over the line length limit or something. I dunno, I just don't question what formatting the autoformat feature does for me usually...

Reply:
here's what it shows in the status bar
![image](https://user-images.githubusercontent.com/837032/97421169-3f333a80-191d-11eb-8869-5d2d5397fec9.png)

Reply:
OK, I had to add graphql manually to the prettier config in PHPStorm, now it does the same 👍 
`updateAsset` is simply too long for one line.